modules/skeletonRetriever/app/conf/config.ini

```diff
@@ -1,5 +1,5 @@
 [general]
-period 0.01
+period 0.1
```

Comment:
`period` sets up the calling period of `updateModule` method, which is responsible for several tasks (i.e., there's no callback on the input keypoints and depth images).

Hence, did you already test that selecting the exact rate of the incoming `yarpopenpose` outputs is just fine? Probably, going with `period 0.05` gives a bit of margin.

Reply:
The output seemed fine, but it's indeed better to set it to 0.05, at least until we can ensure that yarpopenpose can run at higher rates.

Reply:
Switched to 0.05 in latest commit